apps/moose-console/README.md

```diff
@@ -0,0 +1,30 @@
+## Getting Started
+
+First, run the development server:
+
+```bash
+yarn dev
```

Comment:
`yarn`?

Should we remove this altogether or fix it

Reply:
probably remove